zappa/zappa.py

```diff
@@ -877,6 +862,59 @@ def deploy_api_gateway(self, api_id, stage_name, stage_description="", descripti
 
         return "https://{}.execute-api.{}.amazonaws.com/{}".format(api_id, self.boto_session.region_name, stage_name)
 
+    def create_api_key(self, api_id, stage_name):
+        """
+        Create new API key
+        """
+        try:
+            print("Creating API Key..")
+            response = self.apigateway_client.create_api_key(
+                name='{}_{}'.format(stage_name, api_id),
+                description='Api Key for {}'.format(api_id),
+                enabled=True,
+                stageKeys=[
+                        {
+                            'restApiId': '{}'.format(api_id),
+                            'stageName': '{}'.format(stage_name)
+                        },
+                    ]
+                )
+            print('x-api-key: {}'.format(response['id']))
+        except botocore.client.ClientError as e:
```

Comment:
I'd say that unless this function is returning an expected value that will be checked, that we actually _don't_ try/catch this error. Same goes for remove.